CMakeLists.txt

```diff
@@ -27,8 +27,23 @@ enable_testing()
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-# torch, for which we are going to rely on the system for now
-find_package(Torch REQUIRED)
+# Location of current CPU torch
+include(FetchContent)
+set(FETCHCONTENT_BASE_DIR "${CMAKE_SOURCE_DIR}/extern")
+FetchContent_Declare(
+  torch
+  URL https://download.pytorch.org/libtorch/nightly/cpu/libtorch-shared-with-deps-latest.zip
+)
```

Comment:
I'm fine with building against nightly build. If so, I'd recommend scheduling a github action to run nightly as well. This should be simply
```
on:
  schedule:
    - cron: '0 0 * * *'
```